docs/source/templates/layout.html

```diff
@@ -31,7 +31,7 @@
     to release by installing from GitHub:
   </p>
   <p>
-    <code>pip install https://github.com/alteryx/featuretools/archive/woodwork-integration.zip</code>
+    <code>pip install featuretools==1.0.0rc1</code>
```

Comment:
@rwedge do we need to change this to (or no?):
```suggestion
    <code>pip install --pre featuretools==1.0.0rc1</code>
```

Reply:
Do we need the `--pre` flag since we are specifying the release candidate version here? Or should this just be `pip install --pre featuretools`? Not clear on how that all works.

Reply:
If we want to use `--pre` it would just be `pip install --pre featuretools`

If we do `pip install featuretools==1.0.0rc1`, featuretools should be the only pre-release package that gets used

Reply:
Alright let's keep this as is.

Reply:
I think the `--pre` install option could be more in a tweet / emailed instructions since it will grab more recent pre-releases if we end up doing multiple pre-releases, but in the docs we can update it as we go so not as useful here